sky/shell/platform_view.h

```diff
@@ -77,11 +77,14 @@ class PlatformView {
 
   std::unique_ptr<Rasterizer> rasterizer_;
   std::unique_ptr<Engine> engine_;
+  SkyEnginePtr engine_ptr_;
```

Comment:
Never mind. @abarth tells me that @johnmccutchan is unblocked. Let me know if that is not the case. Closing.